### PR TITLE
ci: add release pipeline

### PR DESCRIPTION
**Pull request by xoxys**

### CODE REVIEW

**Comment by xoxys** [comment]
@d7oc As always, please add any feedback. I'm going to merge it already as this change need to be finally tested on main.

File: charts/owncloud/values.yaml [hunk 1]
```diff
@@ -25,20 +25,6 @@ owncloud:
   volume_files: /mnt/data/files
   volume_root: /mnt/data
   configExtra: {}
-  # configExtra:
```
Comment (d7oc) [inline charts/owncloud/values.yaml:28]: Would be good if we could keep this as example

Comment (xoxys) [reply charts/owncloud/values.yaml:28]: We have documented it now dont like to duplicate it.

Comment (d7oc) [reply charts/owncloud/values.yaml:28]: Ok

**Pull request merged**